client/src/components/GameArea.js

```diff
@@ -289,7 +258,9 @@ class GameArea extends Component {
             >
               <Row className="sidebar-col-container-row mh-100 h-100 mw-100 w-100 p-0 m-0">
                 <Col className="side-bar-col-container-row-col w-100 mw-100 flex-shrink-3 p-0 m-0">
-                  <Sidebar />
+                  <Sidebar
+                    cards={cards}
```

Comment:
Do the cards in the sidebar just match those on the battlefield now then? We can merge this as is and update it later if changing that is complicated, just checking if that is right here.

Reply:
Yeah, at this point `GameArea` does no sorting of cards and just displays all the cards in the Cards collection on the battlefield. It would be a quick change to show just cards that are actually on the battlefield on the battlefield so I'll push another commit with that before merging. `Sidebar` gets the full list of cards and filters them as necessary before passing them to each `SideZone` component.